README.md

```diff
@@ -25,24 +25,10 @@ Add the following to your `~/.emacs` or `~/.emacs.d/init.el` file:
 ;; Reason setup
 ;;----------------------------------------------------------------------------
 
-(defun chomp-end (str)
-  "Chomp tailing whitespace from STR."
-  (replace-regexp-in-string (rx (* (any " \t\n")) eos)
-                            ""
-                            str))
-
 (defun shell-cmd (cmd)
   "Returns the stdout output of a shell command or nil if the command returned
    an error"
-  (let ((stdoutput (chomp-end
-                    (with-output-to-string
-                      (with-current-buffer
-                          standard-output
-                        (process-file shell-file-name nil
-                                      '(t nil)  nil
-                                      shell-command-switch cmd))))))
-    (when (not (= (length stdoutput) 0))
-      stdoutput)))
+  (car (ignore-errors (apply 'process-lines (split-string cmd)))))
```

Comment:
Way better, but maybe we don't want to ignore-errors?

Reply:
It's exactly what the previous version did though. We wanna return `nil`

Reply:
Saw that below yes